### PR TITLE
Add config arguments for study documentation

### PR DESCRIPTION
**Pull request by kelshmo**
I added arguments to customize the example study descriptions that are provided in the Documentation tab.

### CODE REVIEW

File: R/mod-upload-documentation.R [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #' UI function for the upload_documentation module
 #' @param id the module id
 #' @return html ui for the module
-upload_documents_ui <- function(id) {
+upload_documents_ui <- function(id, study_link_human, study_link_animal) {
```
Comment (karawoo) [inline R/mod-upload-documentation.R]: Could you please document the new parameters in the roxygen comments above the function and then run `devtools::document()`?

**Comment by karawoo** [comment]
Thanks @kelshmo, can you resolve the merge conflict? I can help if needed.

**Comment by kelshmo** [comment]
@karawoo merge conflict should be corrected.

**Comment by karawoo** [comment]
Thanks! I'll let the build complete before merging.

**Pull request merged**